### DISCUSSION TITLE
Hope YaPa can continue to be developed.

### DISCUSSION THREAD

**Discussion by BlackBeau211** [YetAnotherPomodoroApp/YAPA-2#108] [Ideas]
Firstly, I just want to thank all of the contributors, for me yapa is so helpful and awesome. But in the future, may I can change the digital clock to some moving images such as a sand timer, a circular countdown clock. P/S: Sorry my English is bad.

**Comment by floatas** [comment]
Thank you, I really appreciate the feedback. When working on an open source project and knowing that someone is using it on a daily basis is really motivating. 
I tried to implement different types of clock interfaces, but I realized that I don't know WPF that well (technology used in YAPA).

**Reply by SwapnilChand** [reply]
been using it for 4/5 years now. always handy to lock in and just get some coding done.
